### ISSUE TITLE
Regression in v17.3.8 (marked-terminal peer dependency)

### ISSUE THREAD

**Issue by thislooksfun** [semantic-release/semantic-release#1784]
## Current behavior
When installing semantic-release v17.3.8 nom outputs a warning that `marked-terminal` wants a peer dependency of `marked < 2`, but 17.3.8 updated `marked` to v2.0.0.

## Expected behavior
It should not warn about incompatible package versions.

## Environment

- **semantic-release** version: 17.3.8
- CI environment: N/A
- Plugins used: N/A
- **semantic-release** configuration: N/A
- CI logs: N/A

**Comment by felipecrs** [comment]
This needs to be done at https://github.com/mikaelbr/marked-terminal/pull/64.

**Comment by thislooksfun** [comment]
mikaelbr/marked-terminal#64 was merged into master 2 hrs ago, just waiting on a new release now.

**Comment by thislooksfun** [comment]
Progress towards the new release is being tracked in mikaelbr/marked-terminal#66

**Comment by thislooksfun** [comment]
marked-terminal v4.1.1 has been released which solves this issue.

**Comment by gr2m** [comment]
could you send a PR to bump the minimal version of `marked-terminal` to v4.1.1?

**Comment by thislooksfun** [comment]
@gr2m Just opened #1829.
